mirrord/layer/tests/common/mod.rs

```diff
@@ -702,6 +702,7 @@ pub enum Application {
     EnvBashCat,
     NodeFileOps,
     NodeSpawn,
+    NodeInterfaceAddrs,
```

Comment:
Maybe change the name of the app to the issue (we have quite a few like that).

`NodeIssue2903`.